lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -1,3 +1,4 @@
+require 'active_model_serializers/json_api_objects'
```

Comment:
Arguably those are part of the JsonApi adapter. I don't feel strongly about it, but I'd rather be on the conservative side unless there's a good reason.